plugins/polio/api/vaccines/vaccine_authorization.py

```diff
@@ -247,6 +253,18 @@ def get_most_recent_authorizations(self, request):
 
                 response.append(vacc_auth)
 
+        if auth_status:
+            response = [entry for entry in response if entry["status"] in auth_status.split(",")]
+
+        if block_country:
+            block_country = block_country.split(",")
+            block_country = Group.objects.filter(pk__in=block_country)
+            org_units = [country.org_units.all() for country in block_country]
+            ou_pk_list = []
+            for ou_q in org_units:
+                ou_pk_list = [ou.pk for ou in ou_q]
```

Comment:
Something like this would be more efficient and more readable (I think) and will avoid duplicated pks
```python
import itertools
org_units_ids = [ country.org_units.all().values_list('pk', flat=True) for coutry in block_country ]
ou_pk_list = set(itertools.chain.from_iterable(org_units_ids))
```
If you are sure there will be no duplicates you can use `list` instead of `set`

```

Reply:
I already simplified it, no ?